### ISSUE TITLE
Launch url-replacement-v2

### ISSUE THREAD

**Issue by lannka** [ampproject/amphtml#13371]
A couple of things need to be done before we can fully turn on the feature flag.

- [x] migrate sync calls to use v2 code
- [x] handle opt_collectVars (or remove)
- [ ] documentation

After turning on the feature flag, there will be clean up work to remove v1 code.

**Comment by rudygalfi** [comment]
@calebcordry Could you provide an update?

**Comment by adamkasprowicz** [comment]
Hey @lannka,
Is there any good news?

**Comment by carolinephilbin** [comment]
Hey @lannka @calebcordry what's the status here? Any updates?

**Comment by lannka** [comment]
@carolinephilbin the work is done. however, last week we caught a [bug](https://github.com/ampproject/amphtml/issues/16917) when we try to roll it out. 

The bug has been fixed in canary. It will land in prod next Tuesday (8/7/2018). 

After that, we'll update documentation.

**Comment by lannka** [comment]
Re base64: @adamkasprowicz thanks for foreseeing the issue
Opened a separate bug for that: #17282

**Comment by lannka** [comment]
> The bug has been fixed in canary. It will land in prod next Tuesday (8/7/2018).

Although we still plan to take 1-2 weeks to gradually ramp up the traffic to v2. So the full availability of the feature will be around (8/14/2018 or 8/21/2018)

**Comment by carolinephilbin** [comment]
@lannka great, thanks for the update!

**Comment by carolinephilbin** [comment]
@lannka will the full availability of this feature allow for a successful Mixpanel integration?

**Comment by lannka** [comment]
Yes, if what you need is Base64 and Jason.stringify.

We will try to fix the base64 unicode issue before that

**Comment by adamkasprowicz** [comment]
Hi @lannka! What's the news?
I'm watching 2 issues - this one, and the other one related to XHR body (#15729). Merging any of them  is enough for us to be able to integrate with AMP.
Thanks!

**Comment by lannka** [comment]
This is already fully turned on in production. We're missing documentation thought.

**Comment by adamkasprowicz** [comment]
@lannka Thank you! 🍾 one thing that is missing is JSON.stringify - right now we can't use it, because of this condition:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/variables.js#L195

To replicate the issue:
```javascript
{
  "requests": {
    "pageview": "http://someurl.com/?data=$JSON(${someObject})"
  },
  "vars": {
    "someObject": {"x": 123, "y": 345},
    "someArray": [1, 2, 3],
    "someVar": 444
  },
  "triggers": {
    "trackPageview": {
      "on": "visible",
      "request": "pageview"
    }
  },
  "transport": {
    "beacon": false,
    "xhrpost": false,
    "img": true
  }
}
```

Result:
data=%22%255Bobject%2520Object%255D%22
(it's just "Object", without properties etc)

Expected:
data=%22%7B%22x%22%3A123%2C%22y%22%3A345%7D%22

**Comment by lannka** [comment]
@adamkasprowicz thanks for reporting. created a separate issue to track. #17657

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @calebcordry Do you have any updates?

**Comment by wgins** [comment]
Are there any updates to this @calebcordry ?

**Comment by calebcordry** [comment]
@wgins This should be available for use. Please let me know if you run into any issues.

**Comment by wgins** [comment]
Super. Thanks!
